api/app/settings/common.py

```diff
@@ -141,9 +141,19 @@
 
 SITE_ID = 1
 
+DJANGO_DB_CONN_MAX_AGE = 60
+try:
+    DJANGO_DB_CONN_MAX_AGE = env.int("DJANGO_DB_CONN_MAX_AGE")
+except ValueError:
+    DJANGO_DB_CONN_MAX_AGE = None
```

Comment:
This will set it none if it's not defined. I think we want the default value to be 60?

Reply:
ah yes perfect - yes default to 60 too

Reply:
I will push

Reply:
If I set 

```
export DJANGO_DB_CONN_MAX_AGE=None
```

I get

```
Traceback (most recent call last):
  File "/Users/ben/projects/flagsmith/api/manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/ben/projects/flagsmith/api/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/ben/projects/flagsmith/api/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 325, in execute
    settings.INSTALLED_APPS
  File "/Users/ben/projects/flagsmith/api/.venv/lib/python3.9/site-packages/django/conf/__init__.py", line 79, in __getattr__
    self._setup(name)
  File "/Users/ben/projects/flagsmith/api/.venv/lib/python3.9/site-packages/django/conf/__init__.py", line 66, in _setup
    self._wrapped = Settings(settings_module)
  File "/Users/ben/projects/flagsmith/api/.venv/lib/python3.9/site-packages/django/conf/__init__.py", line 157, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/Users/ben/.asdf/installs/python/3.9.0/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/ben/projects/flagsmith/api/app/settings/local.py", line 3, in <module>
    from app.settings.common import *
  File "/Users/ben/projects/flagsmith/api/app/settings/common.py", line 144, in <module>
    DJANGO_DB_CONN_MAX_AGE = env.int("DJANGO_DB_CONN_MAX_AGE", 60)
  File "/Users/ben/projects/flagsmith/api/.venv/lib/python3.9/site-packages/environs/__init__.py", line 94, in method
    raise EnvValidationError(
environs.EnvValidationError: Environment variable "DJANGO_DB_CONN_MAX_AGE" invalid: Not a valid integer.
```

Reply:
Because you're trying to set the environment variable to a string. Not sure what you're expecting @dabeeeenster ?

Reply:
> https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-CONN_MAX_AGE:
> 
> The lifetime of a database connection, as an integer of seconds. Use 0 to close database connections at the end of each request — Django’s historical behavior — and None for unlimited persistent connections.

How would you specify None as the env var?

Reply:
Wait, i am  looking into it

Reply:
The only way to do that, realistically, would be to either: 

1. Set `None` as the default (not sure why we have `60` as the default?)
2. Use e.g. `-1` as a special value and do something like: 

```
db_connection_max_age = env.int("DJANGO_DB_CONN_MAX_AGE", default=60)
DJANGO_DB_CONN_MAX_AGE = (
    None if db_connection_max_age == -1 else db_connection_max_age
)
```

Reply:
Yeah maybe go with `-1`?

Reply:
yeah, -1 seems like the only solution, unless we want to complicate parsing(which is not worth it)